lms/static/sass/multicourse/_courses.scss

```diff
@@ -125,8 +157,214 @@
     color: $white;
     text-shadow: none;
 
+    //STATE: hover, focus
     &:hover, &:focus {
       background: $m-blue-l1;
     }
   }
+
+  .filters {
+    @include clearfix();
+    margin-top: ($baseline/2);
+    border-top: 1px solid $courseware-button-border-color;
+    border-bottom: 1px solid $courseware-button-border-color;
+    width: 100%;
+    height: auto;
+    max-height: ($baseline*10);
+
+    ul {
+      @include padding-left(0);
+      margin: 0;
+      list-style: outside none none;
+    }
+
+    li {
+      @include float(left);
+      @include margin(($baseline/2), $baseline, ($baseline/2), 0);
+      padding: ($baseline/2) ($baseline*0.75);
+      position: relative;
+      background: $courseware-button-border-color;
+      width: auto;
+
+      a {
+        @extend %t-strong;
+        color: $gray-d2;
+        text-decoration: none;
+
+        i {
+          color: $gray-l2;
+        }
+      }
+    }
+
+    .clear-filters {
+      @include line-height(29.73);
+      @extend %t-icon5;
+      @extend %t-strong;
+      margin: ($baseline/2) 0;
+      width: auto;
+      text-align: center;
+      color: $m-blue-d1;
+    }
+
+    .flt-right {
+      @include float(right);
+    }
+  }
+
+
+
+  .search-facets{
+    @include rtl() { $layout-direction: "LTR"; }
+    @include ltr() { $layout-direction: "RTL"; }
```

Comment:
I think that a comment would be in order here, explaining why these are needed here.